packages/frontend/components/composed/journal/JournalHero/JournalHero.styles.ts

```diff
@@ -1,6 +1,30 @@
 import styled from "styled-components";
+import { pxToRem } from "@wdp/lib/theme/functions";
 
 export const HeroInner = styled.div`
   padding-block-start: var(--container-padding-lg);
   padding-block-end: var(--container-padding-lg);
 `;
+
+export const TitleBlock = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: ${pxToRem(670)};
+`;
+
+export const Title = styled.h2`
+  margin-block-end: ${pxToRem(16)};
```

Comment:
@1aurend There are some common padding values that are defined in root: https://github.com/NGLPteam/wdp-client/blob/main/packages/frontend/theme/base/root.ts#L159-L164
```
/* padding */
    --padding-lg: ${pxToRem(32)};
    --padding-md: ${pxToRem(24)};
    --padding-rg: ${pxToRem(16)};
    --padding-sm: ${pxToRem(8)};
    --padding-xs: ${pxToRem(6)};
```
I wish I had made these in admin! Let's use these variables where we can.